web/tests/views/test_importer_views.py

```diff
@@ -1,86 +1,76 @@
 import os
-from typing import Optional
+
+from django.contrib.auth.models import Group
```

Comment:
This is not used in the file. Any reason why you import it?

Reply:
done